core/tests/performance_tests/splash_test.py

```diff
@@ -0,0 +1,31 @@
+"""Tests for the splash page"""
+
+import unittest
+from test_utils import TestBase
+
+
+class SplashPageTest(TestBase):
+    """Tests for `splash page`."""
+
+    # how do I call setUp only once before all the test cases.
```

Comment:
There's [setUpClass](https://docs.python.org/2/library/unittest.html#unittest.TestCase.setUpClass) (which I found in a few seconds just by googling; it was the top result for "python call setup once"). Having said that, I'm not super keen on introducing a new pattern; it seems more in line with existing tests to consolidate total size and load time into a single test. We already think of individual tests as being independent; there's no need to break that mental model.